modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -515,69 +774,101 @@
      * @deprecated Since version 2.8.
      */
     @Deprecated
+    @SystemProperty("Force all SQL queries to be processed lazily regardless of what clients request")
     public static final String IGNITE_SQL_FORCE_LAZY_RESULT_SET = "IGNITE_SQL_FORCE_LAZY_RESULT_SET";
 
     /** Disable SQL system views. */
+    @SystemProperty("Disables SQL system views")
     public static final String IGNITE_SQL_DISABLE_SYSTEM_VIEWS = "IGNITE_SQL_DISABLE_SYSTEM_VIEWS";
 
     /** SQL retry timeout. */
+    @SystemProperty(value = "SQL retry timeout in milliseconds", type = Long.class, defaults = "30 seconds")
     public static final String IGNITE_SQL_RETRY_TIMEOUT = "IGNITE_SQL_RETRY_TIMEOUT";
 
     /** Enable backward compatible handling of UUID through DDL. */
+    @SystemProperty("Enables backward compatible handling of UUID through DDL")
     public static final String IGNITE_SQL_UUID_DDL_BYTE_FORMAT = "IGNITE_SQL_UUID_DDL_BYTE_FORMAT";
```

Comment:
Should we consider moving the default value to the constant like for other constant does?
```
IgniteSystemProperties.getBoolean(IgniteSystemProperties.IGNITE_SQL_UUID_DDL_BYTE_FORMAT, false);
```

Reply:
`false` is the default value for `IgniteSystemProperties#getBoolean` and for `SystemProperty#defaults` annotation.

1. This param is redundant here.
2. Constant not required.